tests/framework/interface.go

```diff
@@ -30,7 +30,8 @@ type testRunner interface {
 
 type Cluster interface {
 	Members() []Member
-	Client() Client
+	Client(cfg clientv3.AuthConfig) (Client, error)
+	MustClient(cfg clientv3.AuthConfig) Client
```

Comment:
There is no need to have a separate method `MustClient` for interface. We can keep the interface simple by having a normal function.

Reply:
Done in the next PR revision